phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -950,20 +1004,30 @@ private void verifyIndexRows(List<KeyRange> keys,
         }
         if (keys.size() > 0) {
             for (KeyRange keyRange : keys) {
-                String errorMsg = "Missing index row";
                 byte[] key = keyRange.getLowerRange();
                 List<Mutation> mutationList = indexKeyToMutationMap.get(key);
-                if (mutationList.get(mutationList.size() - 1) instanceof Delete) {
+                Mutation mutation = mutationList.get(mutationList.size() - 1);
+                if (mutation instanceof Delete) {
                     continue;
                 }
+                long currentTime = EnvironmentEdgeManager.currentTime();
```

Comment:
Better to use the Phoenix EnvironmentEdgeManager, though the HBase one would also work.